scripts/create-symlinks.js

```diff
@@ -12,6 +12,7 @@ mkdirp( 'node_modules/@wordpress', () => {
 		[ 'packages/browserslist-config', 'node_modules/@wordpress/browserslist-config' ],
 		[ 'packages/babel-preset-default', 'node_modules/@wordpress/babel-preset-default' ],
 		[ 'packages/jest-preset-default', 'node_modules/@wordpress/jest-preset-default' ],
+		[ 'packages/npm-package-json-lint-config', 'node_modules/@wordpress/npm-package-json-lint-config' ],
```

Comment:
I was wondering if we can create this list using `glob`?

Reply:
Possibly, will follow up in another PR